mmpose/datasets/datasets/top_down/topdown_coco_dataset.py

```diff
@@ -204,6 +204,7 @@ def _load_coco_keypoint_annotation_kernel(self, img_id):
                 'image_file': image_file,
                 'center': center,
                 'scale': scale,
+                'bbox': obj['clean_bbox'][:4],
```

Comment:
Is this a bugfix?

Reply:
To make it compatible with JHMDB dataset.

Reply:
okay